seaborn/distributions.py

```diff
@@ -216,10 +214,7 @@ def distplot(a, bins=None, hist=True, kde=True, rug=False, fit=None,
         if bins is None:
             bins = min(_freedman_diaconis_bins(a), 50)
         hist_kws.setdefault("alpha", 0.4)
-        if LooseVersion(mpl.__version__) < LooseVersion("2.2"):
```

Comment:
The [minimal matplotlib version](https://github.com/mwaskom/seaborn/blob/master/setup.py#L32) is 2.1.2 so this path needs to stick around a little while longer.

Reply:
["density" kw exists in mpl==2.1.2](https://matplotlib.org/2.1.2/api/_as_gen/matplotlib.pyplot.hist.html) as well (and has the same meaning), so this works the same across all supported versions.

Reply:
OK great, they were both around for a while.